### PR TITLE
[#89968596] requested by Sarah

### PR DESCRIPTION
**Pull request by randomwire**
From Sarah:

David, could we remove the "use this, not that" for ampersands (we're using them now) in the Style Guide? Could you also change the first sentence in the Style Guide intro to:

Since our founding in 2008, the web and its technology have changed significantly.

### CODE REVIEW

**Comment by kelvintaywl** [comment]
LGTM :hamburger:

File: docs/_includes/voice/style-guide.html [hunk 1]
```diff
@@ -168,10 +168,6 @@ <h1 id="style-guide">Style guide</h1>
     </thead>
     <tbody>
       <tr>
-        <td>and</td>
-        <td>&</td>
```
Comment (ninotoshi) [inline docs/_includes/voice/style-guide.html]: Shouldn't this be `&amp;`?

**Comment by marexandre** [comment]
LGTM

File: docs/_includes/voice/style-guide.html (hunk 1 above)
Comment (marexandre) [inline docs/_includes/voice/style-guide.html]: Good point, but as it's getting removed it's OK :wink:

Comment (randomwire) [inline docs/_includes/voice/style-guide.html]: It's been deleted!

**Pull request merged**